Gemfile

```diff
@@ -38,6 +38,8 @@ gem 'rails-observers', '~> 0.1'
 gem 'protected_attributes', '~> 1.1.1'
 gem 'sprockets', '~> 3'
 gem 'sprockets-rails', '>= 2.3.3', '< 3'
+# FIXME This needs to be renamed if you like the patch
+gem 'foreman_name_generator', :path => '../foreman-name-generator'
```

Comment:
Yeah, I think it's a good addition in general for Foreman & simple enough that making it a plugin defeats the purpose of providing sane defaults